- Fix libz check
  src.c:4:14: error: use of undeclared identifier 'NULL'
         if (NULL != zzz)
             ^
- Install manpages under ${PREFIX}/man and not ${PREFIX}/share/man

--- src/CMakeLists.txt.orig	2017-11-08 12:25:31 UTC
+++ src/CMakeLists.txt
@@ -156,6 +156,7 @@ endif(NOT LIBC_HAS_ARGP)
 set (CMAKE_REQUIRED_LIBRARIES z)
 check_c_source_compiles (
     "#include <zlib.h>
+     #include <stdlib.h>
      int main(int argc, char **argv) {
          gzFile zzz  = gzopen(\"/dev/null\", \"rb\");
          if (NULL != zzz)
@@ -605,7 +606,7 @@ install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/../desktop/
 install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/../desktop/metainfo/yoshimi.appdata.xml
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/metainfo)
 install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/../desktop/yoshimi.1
-    DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/man/man1)
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/man/man1)
 
 set_directory_properties (PROPERTIES
     ADDITIONAL_MAKE_CLEAN_FILES "${FltkUI_headers}"
